src/components/CarouselPhoto.js

```diff
@@ -0,0 +1,67 @@
+import PhotoPost from './PhotoPost';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css'; 
+import 'slick-carousel/slick/slick-theme.css';
+
+/** A carousel photo component.
+ * 
+ * Example of usage:
+ * ```
+ *  <CarouselPhoto images={['https://interactive-examples.mdn.mozilla.net/media/cc0-images/grapefruit-slice-332-332.jpg',
+ *      'https://upload.wikimedia.org/wikipedia/commons/thumb/b/b6/Image_created_with_a_mobile_phone.png/1200px-Image_created_with_a_mobile_phone.png',
+ *      'https://im0-tub-ru.yandex.net/i?id=84dbd50839c3d640ebfc0de20994c30d&n=27&h=480&w=480']}>
+ *  </CarouselPhoto>
+ * ```
+ * 
+ * @param {*} images Array of link to image sources.
+ * @returns carousel view.      
+ */
+const CarouselPhoto = ({ images }) => {
+    const settings = {
+        dots: true,
+        infinite: true,
+        speed: 500,
+        slidesToShow: 1,
+        slidesToScroll: 1,
+        prevArrow: <SamplePrevArrow/>,
+        nextArrow: <SampleNextArrow/>,
+    };
+
+    const style = {
+        width: '625px',
+        height: '100%'
+    };
+
+    return (<Slider {...settings} style={style}>
+        {images.map((image, index) => {
+            return (<PhotoPost image={image} key={index}></PhotoPost>);
```

Comment:
https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318 
index z tablicy nie powinien być kluczem